### DISCUSSION TITLE
Troubleshoot DNS-over-HTTPS Mobileconfig (iPhone)

### DISCUSSION THREAD

**Discussion by attinderdhillon** [AdguardTeam/AdGuardHome#3199] [General]
DNS-over-TLS in mobileconfig works but DoH is not working. DoH link can be opened in the browser, so can't be a network issue.

DoH works in firefox and also tested with dnslookup tool.

Kindly help me troubleshoot why DoH not working with mobileconfig.

**Comment by szhu25** [comment]
I guess you should at least share your Mobile config file?
